.gitignore

```diff
@@ -29,4 +29,4 @@ core
 /.cproject
 /.project
 /tags
-/TAGS
+/TAGS
```

Comment:
revert the change

Reply:
I already fix it, and did a force push.

Reply:
Just saw the new change, the PR looks fine now.